ajax/libs/jquery.tiptip/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.tiptip",
+  "description": "Intelligent ToolTip jQuery Plugin",
+  "homepage": "https://github.com/drewwilson/TipTip",
+  "keywords": [
+    "jquery",
+    "tiptip",
+    "tooltip"
+  ],
+  "version": "1.3",
+  "licences": [
+    "MIT",
+    "GPL"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drewwilson/TipTip"
+  },
+  "filename": "README.txt",
```

Comment:
it was .js file, but cdn importer seems have broken it:

https://github.com/glensc/jquery.tiptip/blob/master/package.json#L17